frontend/Views/Widgets/ViewNemesisStats.svelte

```diff
@@ -0,0 +1,312 @@
+<script>
+    import { onMount } from "svelte";
+    import Chart from "chart.js/auto";
+
+    export let viewId;
+    export let dashboardObject;
+    let chartCanvas;
+    let chart;
+    let barChartCanvas;
+    let barChart;
+    let versions = [];
+    let releases = [];
+    let selectedFilter = "";
+    let totalDuration = 0;
+    let uniqueNemesisCount = 0;
+    let allNemesisData = [];
+    let errorMsg = "";
+    let loading = false;
+    let filteredData = [];
+
+    async function fetchData() {
+        try {
+            const response = await fetch(`/api/v1/views/widgets/nemesis_data?view_id=${dashboardObject.id}`);
+            if (!response.ok) {
+                throw new Error(`HTTP error! status: ${response.status}`);
+            }
+            const data = await response.json();
+            allNemesisData = data.response.nemesis_data || [];
```

Comment:
Missing check for API status

Reply:
it is in above `if (!response.ok) {`

Reply:
anyway I noticed even if failure there's 200 response and error details are provided. I adjusted code to use it.